includes/rest/class-outbox.php

```diff
@@ -72,7 +72,7 @@ public static function user_outbox_get( $request ) {
 		$json->generator = 'http://wordpress.org/?v=' . \get_bloginfo_rss( 'version' );
 		$json->actor = \get_author_posts_url( $user_id );
 		$json->type = 'OrderedCollectionPage';
-		$json->partOf = \get_rest_url( null, "/activitypub/1.0/users/$user_id/outbox" ); // phpcs:ignore
+		$json->partOf = get_rest_url_by_path( sprintf( 'users/%d/outbox', $user_id ) ); // phpcs:ignore
```

Comment:
`get_rest_url_by_path` is in the `Activitypub` namespace, so you have to add `use function Activitypub\get_rest_url_by_path` as the top of the class.